### DISCUSSION TITLE
支付宝支付，申请退款

### DISCUSSION THREAD

**Discussion by songdh** [getlantern/lantern-forum-cn#991] [General]
通过支付宝两次支付成功，没有开通专业版，申请退款。
第一次 ：09-12 15:38:01  订单号2024091222001481061451500263
绑定邮箱 无
![image](https://github.com/user-attachments/assets/4800da80-da37-4705-8de7-3d703c70b7a4)

第二次：09-12 15:40:54 订单号：2024091222001481061448564570
绑定邮箱 [songdh.dream@gmail.com](mailto:songdh.dream@gmail.com)
![image](https://github.com/user-attachments/assets/b2348c68-1e73-4d9d-bba8-87b2b8d434da)

**Comment by MaxMKMax** [comment]
@songdh 您好. 已经帮您提交上去了.

**Comment by crossterra-agent** [comment]
- **Status**: DONE
- **Role**: orchestrator
- **Step**:
- **Link**: https://github.com/CrossTerra/monorepo/pull/1923
- **Summary**: Duplicate invocation — CI checks still failing after D06; no new ci-watcher status received; waiting for ci-watcher to invoke the developer to fix failing pytest jobs.
